src/style.css

```diff
@@ -1 +1,27 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(1n) {
```

Comment:
The selector `.stars--5 :nth-child(1n)` is equivalent to selecting all children. If the intention is to select all stars, this is correct, but if you meant to select a specific number, consider revising this selector.